ajax/libs/jquery-endless-scroll/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery-endless-scroll",
+  "filename": "jquery.endless-scroll.min.js",
+  "version": "1.8.0",
+  "description": "Endless/infinite scrolling/pagination.",
+  "keywords": [
+    "jquery",
+    "endless",
+    "scroll"
+  ],
+  "licenses": [
+    "MIT",
+    "GPL"
```

Comment:
You should add the vision of the GPL license. This [lib page](https://github.com/fredwu/jquery-endless-scroll/#license) provides the version of the GPL license.
Ref: https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json